Library/Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,33 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "http://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "http://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+
+  depends_on "cmake" => :build
+  depends_on "gstreamer"
+  depends_on "glib"
+  depends_on "gst-plugins-base"
+  depends_on "boost"
+  depends_on "doxygen" => :optional
```

Comment:
I suspect this is only needed at build time. If that's the case: `[:optional, :build]`